### ISSUE TITLE
Use of this repo

### ISSUE THREAD

**Issue by dacrome** [osiam/osiam#1]
What should this repo contain?
- documentation
  - What about the https://github.com/osiam/osiam.github.io repo? Or should the io page wrap all the documentation together to website?
- distribution
  - Maybe the https://github.com/osiam/distribution repo content should move here?
- umbrella project
  - List of current OSIAM projects and it's versions

Feel free to comment!

**Comment by dacrome** [comment]
Maybe also use this repo for discussions instead of the [OSIAM google groups forum](https://groups.google.com/forum/#!forum/osiam)? Or better use [gitter](https://gitter.im/)?

**Comment by tkrille** [comment]
This repo is thought to be there for the following things:
- as main entrypoint for everyone who starts looking into OSIAM
- hosting the main, project-global documentation
- hosting the project-global, technical documentation
- as umbrella project, defining the main OSIAM version and the version of it's components
- as a first place to create issues for bugs, feature requests, etc.
- creation of the main distributions (archive, debian packages, Docker image?) - but this might be better contained in the individual repos!?!?

Some of this will surely be moved to the OSIAM GitHub pages sometimes.

**Comment by tkrille** [comment]
AFAIC, we should still use the Google groups forum.

**Comment by wallner** [comment]
I think for discussions we should stick to the Google group and consider a
real time chat. I'd suggest a freenode irc chat but gitter is probably good
too.

Thomas Krille notifications@github.com schrieb am Mo., 4. Mai 2015 um
10:25 Uhr:

**Comment by tkrille** [comment]
IRC sounds cool. Could be a nice place to discuss things openly. Much better than our internal Jabber or Google Hangouts.

**Comment by tkrille** [comment]
@wallner Do you have any experience getting a fixed channel on freenode?

**Comment by tkrille** [comment]
@dacrome @wallner Is this question now fully answered? I think we should move discussions about IRC, Google Groups to their dedicated issues, if there's still a need to discuss.

**Comment by dacrome** [comment]
Yes, but I think this issue should only be closed when the good parts are moved to the documentation of this repo?

**Comment by tkrille** [comment]
I don't know. Isn't the existence of this issue documentation enough? Where do you want to put it? This repo is there to describe OSIAM and not to describe itself. I think this question is fully answered and everybody understands what to do here. But I'm open for any other thoughts.

**Comment by dacrome** [comment]
Maybe you are right. I just thought some points of [your answer](https://github.com/osiam/osiam/issues/1#issuecomment-98631922) could be moved to the README...

**Comment by tkrille** [comment]
I think I just understood your point. We're having discussions here that produce real decisions and targets for further evolution of OSIAM and standards for how we manage the project, etc. When we found a consensus then the results are fading away in the pool of closed issues. But we cannot, for example, put the port mappings table in the current documentation, because it's the future, not the present time. I also don't want to put the decisions made in this specific issue here in the `README.md`, because this file is for people coming from "the outside" that just want to have a look at or use OSIAM. They don't care what we decided to use this repo for. But I noticed, that the stuff we decide here doesn't even need to be under version control or something, so I re-enabled the wiki feature. I think the wiki is the best place to write all this aforementioned stuff down.

**Comment by dacrome** [comment]
I close this issue as resolved, because I think it's clear for what this repo should be used. We don't need a wiki to document this.
tl;dr: Use GitHub issues instead of google groups forum for questions, IRC/Gitter/WhatEverChat can not be managed by the OSIAM team and everything else what @tkrille wrote [here](https://github.com/osiam/osiam/issues/1#issuecomment-98631922), except the last point (creation of the main distributions), which should be discussed in a new issue.
